_includes/head.html

```diff
@@ -14,6 +14,7 @@
   <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
   <link rel="manifest" href="/site.webmanifest">
   <script defer type="text/javascript" src="/assets/js/lazysizes.min.js"></script>
+  <script defer type="text/javascript" src="/assets/js/unveilhooks.min.js"></script>
```

Comment:
What's the difference between `async` and `defer` for `<scripts>`? Never learnt that :)

Reply:
happy to share! when I was reading up on that I ran into this cool article: https://flaviocopes.com/javascript-async-defer/
basically async fetches the script asynchronously and when it's done doing that it executes it before waiting for the HTML parsing to complete. defer fetches async but executes the script after the HTML is parsed. so defer wins in most cases.
here you've got charts that show this nicely: https://flaviocopes.com/javascript-async-defer/#performance-comparison